src/assets/scss/_input-text.scss

```diff
@@ -53,4 +53,10 @@
       background: url(~@img/icon/icon-hide.svg?url) no-repeat center center;
     }
   }
+
+  input[type='number']::-webkit-inner-spin-button,
+  input[type='number']::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
```

Comment:
Dúvida, temos a confirmação que podemos remover de forma global?

Se não, acho melhor criar uma prop para quando for necessário remover

Reply:
Prop criada! https://github.com/solfacil/components-ui/pull/119/commits/353ccc4255b981e9dc4381eab71a67438dab8f33